rubrik_cdm/data_management.py

```diff
@@ -2062,9 +2060,7 @@ def get_sql_db(self, db_name=None, instance=None, hostname=None, availability_gr
             if ((key == 'limit') or (key == 'offset')) and not isinstance(value, int):
                 raise InvalidParameterException('The limit and offset paremeter must be an integer')
 
-        query = ''
-        for key, value in parameters.items():
-            query = query + ("{}={}".format(key, value) + '&')
+        query = '&'.join(['%s=%s' % kv for kv in parameters.items()])
```

Comment:
Can we add a comment describing what this line is doing?

Reply:
Added comments for the string join

New code in: cf7e991